src/store/modules/Logs/DumpsStore.js

```diff
@@ -146,14 +150,14 @@ const DumpsStore = {
           const errorMsg =
             error.response.data.error?.['@Message.ExtendedInfo'][0].MessageId;
 
-          switch (errorMsg) {
-            case 'Base.1.8.1.ResourceInUse':
+          switch (true) {
```

Comment:
Here also, Why true?

Reply:
I understand that adding true will allow us to condition checks in case unlike using string. This change makes sense.